### PR TITLE
python3Packages.tatsu: 5.5.0 -> 5.6.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
build is broken on staging-next

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117223` at d8646dae run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.ics</li>
    <li>python38Packages.tatsu</li>
    <li>python39Packages.ics</li>
    <li>python39Packages.tatsu</li>
  </ul>
</details>

File: pkgs/development/python-modules/tatsu/default.nix [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
-, dataclasses, typing
-, pytestrunner, pytest-mypy
+, colorama, regex
+, pytestrunner, pytestCheckHook, pytest-mypy
 }:
 
 buildPythonPackage rec {
-  pname = "TatSu";
-  version = "5.5.0";
+  pname = "tatsu";
```
Comment (primeos) [inline pkgs/development/python-modules/tatsu/default.nix:7]: Out of curiosity: Should one always use lowercase / the module name here? (https://github.com/neogeny/TatSu/blob/aec01f6478a37c69612d09050d7228ca3c7141bd/setup.py#L5 and https://github.com/neogeny/TatSu/blob/aec01f6478a37c69612d09050d7228ca3c7141bd/setup.py#L19)

**Review by primeos — APPROVED** [review]
LGTM and thanks for the overdue cleanup!

File: pkgs/development/python-modules/tatsu/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/tatsu/default.nix:7]: Yes, see https://github.com/NixOS/nixpkgs/pull/116344.

**Pull request merged**